From 0b189922414ca879fb029c2bb11ad67e6a7e1395 Mon Sep 17 00:00:00 2001
From: Naveen Krishna Chatradhi <nchatrad@amd.com>
Date: Mon, 23 May 2022 01:47:27 -0400
Subject: [PATCH] apml_modules: add "WITH Linux-syscall-note" to SPDX tag of
 uapi headers

UAPI headers licensed under GPL are supposed to have exception
"WITH Linux-syscall-note" so that they can be included into non-GPL
user space application code.

The exception note is missing in amd-apml.h UAPI header.

Signed-off-by: Naveen Krishna Chatradhi <nchatrad@amd.com>
---
 amd-apml.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/uapi/linux/amd-apml.h b/include/uapi/linux/amd-apml.h
index cb8f6b2..0416109 100644
--- a/include/uapi/linux/amd-apml.h
+++ b/include/uapi/linux/amd-apml.h
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note */
 /*
  * Copyright (C) 2021-2022 Advanced Micro Devices, Inc.
  */
--
2.27.0

